action.yml

```
@@ -69,6 +69,35 @@ runs:
+        for FILE in "${added_modified_files[@]}"; do
+            EXT="${FILE##*.}"
+            echo "Changed file: " $FILE ", extension: " $EXT
+            if [[ "$EXT" == "php" && $FILE != migrations/Version* ]]; then
+              echo "src/${FILE}" >> "${FILELIST}"
+              echo "File added to the file list: " $FILE
+            fi
+        done
+        if [ -s "${FILELIST}" ]; then
+            vendor/bin/phpcs --standard=PSR12 --exclude=PSR1.Classes.ClassDeclaration,Squiz.Classes.ValidClassName -v --file-list="${FILELIST}"
+            RETVAL=$?
+            echo "The following files were tested for PSR-12 errors:"; echo 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)